pkg/tsdb/azuremonitor/metrics/azuremonitor-datasource_test.go

```diff
@@ -130,6 +130,39 @@ func TestAzureMonitorBuildQueries(t *testing.T) {
 			expectedInterval:        "PT1M",
 			azureMonitorQueryTarget: "%24filter=blob+eq+%27%2A%27&aggregation=Average&api-version=2018-01-01&interval=PT1M&metricnames=Percentage+CPU&metricnamespace=Microsoft.Compute-virtualMachines&timespan=2018-03-15T13%3A00%3A00Z%2F2018-03-15T13%3A34%3A00Z",
 		},
+		{
+			name: "has dimensionFilter*s* property with not equals operator",
+			azureMonitorVariedProperties: map[string]interface{}{
+				"timeGrain":        "PT1M",
+				"dimensionFilters": []types.AzureMonitorDimensionFilter{{Dimension: "blob", Operator: "ne", Filter: "test"}},
+				"top":              "30",
+			},
+			queryInterval:           duration,
+			expectedInterval:        "PT1M",
+			azureMonitorQueryTarget: "%24filter=blob+ne+%27test%27&aggregation=Average&api-version=2018-01-01&interval=PT1M&metricnames=Percentage+CPU&metricnamespace=Microsoft.Compute-virtualMachines&timespan=2018-03-15T13%3A00%3A00Z%2F2018-03-15T13%3A34%3A00Z&top=30",
```

Comment:
I was testing it manually (pulled in latest changes from main) and after I clicked on View in Azure Portal, I didn't see the dimension filters I applied there. Just wondering if these expected `azureMonitorQueryTarget` implied I was supposed to see them passed through there.

Reply:
Great spot here @fridgepoet, I think this is actually a separate issue in how we build the deep link URL for the query. The URL built above is the URL that is used for data retrieval (the query URL). The URL that is built to link back to the Azure portal does not include filters at the moment, maybe something we can look into implementing.